backend/nodes/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
This isn't how you typically include a license in open source. It's not what source is licensed here, so the license effectively applies to everything in this directory.

It seems like [this](https://stackoverflow.com/questions/1523427/what-is-the-common-header-format-of-python-files) is common practice in python, so we should probably follow it. Just move the licensed function into its own file.

Reply:
Yeah I definitely don't want to accidentally change the license of some of my files